src/appimagetool.c

```diff
@@ -502,6 +502,14 @@ main (int argc, char *argv[])
     char* github_token;
     github_token = getenv("GITHUB_TOKEN");
 
+    /* Parse GitHub CI environment variables.
+     * https://docs.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables
+     */
+    char* GITHUB_REPOSITORY;
```

Comment:
Violates our "code style". Please don't use all-uppercase for local variables. Check e.g. `github_token` as an example how to do it.

Also, you can assign a value directly, no need to split declaration and definition.

Reply:
Fixed on https://github.com/AppImage/AppImageKit/pull/1075/commits/dd7eee51b179f56b01aef7b4f242e090b667343a